filebeat/recipes/default.rb

```diff
@@ -0,0 +1,23 @@
+template '/etc/yum.repos.d/filebeat.repo' do
+  source 'filebeat.repo.erb'
+  owner 'root'
+  group 'root'
+  mode '0755'
+end
```

Comment:
You may need to `depends "yum"` in `metadata.rb` in order to get things from it.